frontend/src/metabase/admin/people/containers/PeopleListingApp.jsx

```diff
@@ -1,11 +1,13 @@
 import PropTypes from "prop-types";
 import { t } from "ttag";
 import { connect } from "react-redux";
+import { Group } from "@mantine/core";
 
 import * as Urls from "metabase/lib/urls";
 
 import AdminPaneLayout from "metabase/components/AdminPaneLayout";
-import Radio from "metabase/core/components/Radio";
+// import Radio from "metabase/core/components/Radio";
+import { Radio } from "metabase/ui/components/inputs/Radio";
```

Comment:
I have a strong preference for  a single top-level export for basic components like this. I shouldn't have to know where a component is to use it.

```suggestion
import { Radio } from "metabase/ui";
```

Reply:
Yup! 100% agree. Thanks for reminding me I still need to do that.